mne/viz/utils.py

```diff
@@ -2287,3 +2287,25 @@ def _ndarray_to_fig(img):
     fig = _figure_agg(dpi=dpi, figsize=figsize, frameon=False)
     fig.figimage(img, resize=True)
     return fig
+
+
+def get_brain_fig(brain):
```

Comment:
This is a new public function. I don’t see it used or tested. Is it actually needed?